docs/topics/exceptions.md

```diff
@@ -1,122 +1,679 @@
 [//]: # (title: Exceptions)
 
-## Exception classes
+Exceptions are key tools for handling runtime errors. They are designed to keep the program running predictably despite encountering 
```

Comment:
🤔 
I think we should specify runtime errors to avoid mudding the line between errors and exceptions
how about this version:
"Exceptions help your code run more predictably, even when runtime errors occur that could disrupt program execution."